diff --git a/pyworkflow/em/packages/opic/protocol_localized.py b/pyworkflow/em/packages/opic/protocol_localized.py
index 6994b5bc21..dac3bcfda7 100644
--- a/pyworkflow/em/packages/opic/protocol_localized.py
+++ b/pyworkflow/em/packages/opic/protocol_localized.py
@@ -189,9 +189,15 @@ class ProtLocalizedRecons(ProtParticlePicking, ProtParticles):
             partItem = pyrelion.Item()
             particleToRow(part, partItem)
             
-            subparticles = self.getSubparticles(localrec, partItem,
-                                                symMatrices, params,
-                                                subpartVectorList, filters)
+            if getVersion() == '1.2.1':
+                subparticles = self.getSubparticles(localrec, partItem,
+                                                    symMatrices, params,
+                                                    subpartVectorList, filters, params["pxSize"])
+            else:
+                subparticles = self.getSubparticles(localrec, partItem,
+                                                    symMatrices, params,
+                                                    subpartVectorList, filters)
+
             for subpart in subparticles:
                 rowToSubcoordinate(subpart, coord, part)
                 coord.setObjId(None) # Force to insert as a new item
@@ -290,8 +296,23 @@ class ProtLocalizedRecons(ProtParticlePicking, ProtParticles):
         return str(maxCounter+1) if maxCounter > 0 else '' # empty if not outputs
     
     def getSubparticles(self, localrec, partItem, symMatrices,
-                        params, subpartVectorList, filters):
-        if getVersion() == '1.1.0':
+                        params, subpartVectorList, filters, pxSize):
+
+        if getVersion() == '1.2.1':
+            subparticles, _ = localrec.create_subparticles(partItem,
+                                                           symMatrices,
+                                                           subpartVectorList,
+                                                           params["dim"],
+                                                           self.randomize,
+                                                           "subparticles",
+                                                           params["unique"],
+                                                           0,
+                                                           self.alignSubparticles,
+                                                           "",
+                                                           False,
+                                                           filters,
+                                                           pxSize)
+        elif getVersion() == '1.1.0':
             subparticles, _ = localrec.create_subparticles(partItem,
                                                            symMatrices,
                                                            subpartVectorList,
@@ -318,5 +339,4 @@ class ProtLocalizedRecons(ProtParticlePicking, ProtParticles):
                                                            "",
                                                            False,
                                                            filters)
-
         return subparticles
